spec/till_spec.rb

```diff
@@ -0,0 +1,40 @@
+require 'till'
+
+describe Till do
+
+  it 'will have list of items' do
+    expect(subject.list.items).to include 'Cappucino'
+  end
+
+  it 'can add items to a list' do
+    subject.add('Americano', 3)
+    expect(subject.order).to include 'Americano'
+  end
+
+  it 'will show error if try ordering nn item not in the list' do
+    expect{subject.add('English Breakfast Tea', 2)}.to raise_error('Not on menu')
```

Comment:
Metrics/LineLength: Line is too long. [81/80] (https://github.com/bbatsov/ruby-style-guide#80-character-limits)